drivers/media/i2c/ov9281.c

```diff
@@ -177,6 +177,7 @@ static const struct regval ov9281_common_regs[] = {
 	{0x372d, 0x22},
 	{0x3731, 0x80},
 	{0x3732, 0x30},
+	{0x3778, 0x00},
```

Comment:
I checked carefully the effect by taking pictures with this register set to 0x10 and to 0x00. In the 1280x* modes it will create lines (I think every second line is grey if that makes any sense). 

In the 640x400 mode it does not do this, but when set, the overall brightness of the picture increases. It can be left at 0x00, but since you found evidence that it is maybe useful, I readded it to that mode. 

I guess you are looking at the same document as me? It is a leaked / scanned datasheet I would say from Innomaker? If yes, I also found the blank part on this register. It seems to be a black box and we cannot be sure if it is required to set it. 

I also found prof that it is used in 640x400 mode by NVidia: https://gitlab.incom.co/CM-Shield/android_kernel_nvidia_quill/-/blob/5ff2f17e8a9293d9f15b1a1de73139260ead1967/drivers/media/i2c/ov9281_mode_tbls.h#L333

But I have a feeling that most of the implementations are done without having the full datasheet…